### DISCUSSION TITLE
how run command hook on windows

### DISCUSSION THREAD

**Discussion by scialo** [garethgeorge/backrest#308] [General]
how can i run comand hook on windows?

**Comment by garethgeorge** [comment]
This is something that probably deserves a feature request for proper support, but at the moment there is a workaround!

You can run a windows command hook by adding a `#! cmd.exe` directive to the top of your script which hints to backrest what process should be used to interpret the script. On windows this will open cmd.exe in a new shell window, it may also misbehave with multiline scripts so I recommend actually placing you script in a file and using a 1 liner to execute it from backrest.

e.g. something similar to 

```
#! cmd.exe
C:\MyBat.bat
```

should work (haven't tested on my Windows machine recently).

**Comment by scialo** [comment]
Thanks. I will try.
